ext/scheduler/airflow/__lib.py

```diff
@@ -69,8 +69,9 @@ def execute(self, context):
 
 
 class OptimusAPIClient:
-    def __init__(self, optimus_host):
+    def __init__(self, optimus_host, timeout):
```

Comment:
this would require changes in other places as well, otherwise this will result in 
`missing required positional argument: 'timeout'`